### ISSUE TITLE
Website still talks about keymapping choices, can't find them in settings

### ISSUE THREAD

**Issue by Lokeh** [cursive-ide/cursive#1952]
It sounds like per https://github.com/cursive-ide/cursive/issues/1863 that this was removed from Cursive, however the [user guide](https://cursive-ide.com/userguide/keybindings.html) still has instructions for it. This was confusing to me.

**Comment by jtlocsei** [comment]
Yeah, I found this confusing in the user guide. If I understand correctly, new users can just skip the part about setting up keybindings?

**Comment by finnjohnsen** [comment]
please just delete the chapter from your docs! I just wasted time tryigng to figure out if I had wrecked my IntelliJ.
